tfjs-backend-cpu/src/utils/kernel_utils.ts

```diff
@@ -15,37 +15,44 @@
  * =============================================================================
  */
 
-import {BinaryInputs, KernelConfig} from '@tensorflow/tfjs-core';
+import {BinaryInputs, KernelFunc} from '@tensorflow/tfjs-core';
 import {DataType, NumericDataType, TypedArray} from '@tensorflow/tfjs-core';
 import {backend_util} from '@tensorflow/tfjs-core';
 
 import {util} from '@tensorflow/tfjs-core';
 import {MathBackendCPU} from '../backend_cpu';
 import {assertNotComplex} from '../cpu_util';
 
-export function createBinaryKernelConfig(
-    name: string,
-    op: (
-        aShape: number[], bShape: number[], aVals: TypedArray,
-        bVals: TypedArray,
-        dtype: DataType) => [TypedArray, number[]]): KernelConfig {
-  return {
-    kernelName: name,
-    backendName: 'cpu',
-    kernelFunc: ({inputs, backend}) => {
-      const {a, b} = inputs as BinaryInputs;
-      const cpuBackend = backend as MathBackendCPU;
-      assertNotComplex([a, b], name);
-
-      const aVals = cpuBackend.data.get(a.dataId).values as TypedArray;
-      const bVals = cpuBackend.data.get(b.dataId).values as TypedArray;
-
-      const [resultData, resultShape] =
-          op(a.shape, b.shape, aVals, bVals, a.dtype);
-
-      const dataId = cpuBackend.write(resultData, resultShape, a.dtype);
-      return {dataId, shape: resultShape, dtype: a.dtype};
-    }
+export type SimpleBinaryOperation = (a: number, b: number) => number;
```

Comment:
nit: might be useful to use this type to replace the type of the "op" parameter in createBinaryKernelImpl below.